src/main/sqldelight/com/jakewharton/dodo/db/tweetIndex.sq

```diff
@@ -0,0 +1,52 @@
+CREATE VIRTUAL TABLE tweet_index USING fts5(
+  status_id INTEGER UNINDEXED,
+  status_text TEXT,
+  status_user_name TEXT,
+  status_user_handle TEXT,
+  retweeted_user_handle TEXT,
+  retweeted_user_name TEXT,
+  quoted_user_handle TEXT,
+  quoted_user_name TEXT,
+  quoted_text TEXT
+);
+
+CREATE TRIGGER tweet_after_insert AFTER INSERT ON tweet BEGIN
```

Comment:
I tested out these triggers by manually inserting garbage data into the tweet database from the command line. It seems to work, but just for sanity sake it might be good if you test this out with your twitter API keys.

Reply:
I should also really write a few sanity-check tests for such cases, but I don't want to spend _too_ much time on this.